test/REQUIRE

```diff
@@ -0,0 +1,8 @@
+julia 0.3
```

Comment:
don't need to duplicate, only list things that are stricter or not included in the top level `REQUIRE`